src/installer/tests/Directory.Build.props

```diff
@@ -10,8 +10,9 @@
     <InternalNupkgCacheDir>$(ArtifactsObjDir)ExtraNupkgsForTestRestore\</InternalNupkgCacheDir>
     <TestArchitectures>$(TargetArchitecture)</TestArchitectures>
     <TestInfraTargetFramework>$(NetCoreAppToolCurrent)</TestInfraTargetFramework>
-    <TestRunnerAdditionalArguments Condition="'$(RunOnStaging)' != 'true'">-notrait category=failing</TestRunnerAdditionalArguments>
-    <TestRunnerAdditionalArguments Condition="'$(RunOnStaging)' == 'true'">-trait category=failing</TestRunnerAdditionalArguments>
+    <TestRunnerAdditionalArguments>-notrait category=failing</TestRunnerAdditionalArguments>
+    <TestRunnerAdditionalArguments Condition="'$(RunOnStaging)' != 'true'">$(TestRunnerAdditionalArguments) -notrait category=FlakyAppHostTests</TestRunnerAdditionalArguments>
+    <TestRunnerAdditionalArguments Condition="'$(RunOnStaging)' == 'true'">$(TestRunnerAdditionalArguments) -trait category=FlakyAppHostTests</TestRunnerAdditionalArguments>
```

Comment:
Doesn't this mean that we only run the FlakyAppHostTests tests?

Reply:
yeah, in runtime-staging. that's how it was before with the `-trait category=failing`.

Reply:
Ah got it. What I'm wondering, do we need both the -notrait category=failing and the -trait category=FlakyAppHostTests when running on staging?

Reply:
I was under the assumption that if you pass in a `-trait ` argument, only those tests are run.

Reply:
probably not, but it doesn't hurt and makes the condition easier to understand IMO

Reply:
right, it shouldn't matter here.